engine/classes/Elgg/Http/ResponseFactory.php

```diff
@@ -144,7 +144,10 @@ public function send(Response $response) {
 						. (string) $this->response_sent);
 			}
 		} else {
-			$response->send();
+			if (elgg_trigger_before_event('send', 'http_response', $response)) {
```

Comment:
My assumption is that the response is always sent, the question is how - using the default method, or i.e. writing to log file.